tests/virt_autotest/login_console.pm

```diff
@@ -224,6 +224,21 @@ sub login_to_console {
     set_ssh_console_timeout_before_use if (is_sle and is_remote_backend and is_x86_64 and get_var('VIRT_AUTOTEST', ''));
     # use console based on ssh to avoid unstable ipmi
     use_ssh_serial_console;
+
+    # Check 64kb page size enabled.
+    if (check_var('KERNEL_64KB_PAGE_SIZE', '64kpg')) {
+        # Verify 64kb page size enabled.
+        record_info('Baremetal kernel cmdline', script_output('cat /proc/cmdline'));
+        assert_script_run("grep -- '-64kb' /proc/cmdline");
```

Comment:
I think content in /proc/cmdline can only guarantee the kernel will be used for booting, but whether a successful boot with 64KB page size should be looked up in dmesg or log files.

Reply:
the parameter from  /proc/cmdline maybe does not take effect. will change into dmesg.